### PR TITLE
Added draft Profile UI

### PR DESCRIPTION
**Pull request by RashikaKarki**

### CODE REVIEW

**Comment by RashikaKarki** [comment]
@eKhattak @Drewbi  Anything I need to add before I merge it?

**Review by anshulrgoyal — COMMENTED** [review]
adding a screenshot will be helpgful :)

**Comment by RashikaKarki** [comment]
image
image
Hey @anshulrgoyal take a look 
Still a lot of things needs to changed design wise but I think it's good to go for connecting with backend for now.

**Comment by Drewbi** [comment]
Could you pull master? The css files we're using are modules and they are called `[name].module.css`

The way you use them is by making a class selection on css as you would normally: 
```css
.profile {
    rule: whatever;
}
```
Then import and use it by passing in `styles.profile` to the elements `className`, eg:
```jsx
import styles from './[name].module.css`
<element className={styles.profile}>Text</element>
```

I'll be honest, I'm not sure how you were using `class` to pass class names as react should prevent you from doing so. Probably will be fixed when the eslint pull request goes through.

**Comment by RashikaKarki** [comment]
@Drewbi Give it a look. Should be good now.

File: client/src/pages/Profile/index.js [hunk 1]
```diff
@@ -1,12 +1,227 @@
...
+  }
+}
+
+class NavItem extends React.Component {
+  render() {
+    return (
+      <li>
+        <span
+          className={classes.item}
+          id={this.props.activeProfile && "active"}
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]: `"active"` won't work when exporting css from `.module.css`. Use `classes.active`

**Review by Drewbi — APPROVED** [review]
Let's merge, I'm sure there will be more changes later

File: client/src/Pages/Profile/index.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                    </div> 
+            </div>
+        )
+    }
+}
+
+class RecentMeet extends React.Component{
+    render(){
+        return(
+            <div className = {classes.card} id = "recentmeeting">
```
Comment (khattakdev) [inline client/src/Pages/Profile/index.js]: Change to `id={classes.recentmeeting}`

File: client/src/Pages/Profile/index.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                {this.state.showRegister && <Popup action = {this.ResheduleInActive}/>}
+            </div>
+        )
+    }
+}
+
+class ProfileHeader extends React.Component{
+    render(){
+        return(
+            <div id="profileheader">
```
Comment (khattakdev) [inline client/src/Pages/Profile/index.js]:
```suggestion
            <div id={classes.profileheader}>
```

File: client/src/pages/Profile/index.js [hunk 4]
```diff
@@ -1,12 +1,227 @@
...
-    </>
-  );
-};
+import "./index.module.css";
+import classes from "./index.module.css";
+
+class Badges extends React.Component {
+  render() {
+    return (
+      <div className={classes.card} id="badge">
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
      <div className={classes.card} id={classes.badge}>
```

File: client/src/pages/Profile/index.js [hunk 5]
```diff
@@ -1,12 +1,227 @@
...
+        </div>
+      </div>
+    );
+  }
+}
+
+class Skills extends React.Component {
+  render() {
+    return (
+      <div className={classes.card} id="skills">
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
      <div className={classes.card} id={classes.skills}>
```

File: client/src/pages/Profile/index.js [hunk 6]
```diff
@@ -1,12 +1,227 @@
...
+    );
+  }
+}
+
+class ProfileContent extends React.Component {
+  render() {
+    return (
+      <div id="profile">
+        <section className={classes.cards}>
+          <div className={classes.card} id="about">
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
          <div className={classes.card} id={classes.about}>
```

File: client/src/pages/Profile/index.js [hunk 7]
```diff
@@ -1,12 +1,227 @@
...
+        </section>
+      </div>
+    );
+  }
+}
+
+class UpcomingMeet extends React.Component {
+  render() {
+    return (
+      <div className={classes.card} id="upcomingmeeting">
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
      <div className={classes.card} id={classes.upcomingmeeting}>
```

File: client/src/pages/Profile/index.js [hunk 8]
```diff
@@ -1,12 +1,227 @@
...
+        </div>
+      </div>
+    );
+  }
+}
+
+class RecentMeet extends React.Component {
+  render() {
+    return (
+      <div className={classes.card} id="recentmeeting">
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
      <div className={classes.card} id={classes.recentmeeting}>
```

File: client/src/pages/Profile/index.js [hunk 9]
```diff
@@ -1,12 +1,227 @@
...
+        {this.state.showRegister && <Popup action={this.ResheduleInActive} />}
+      </div>
+    );
+  }
+}
+
+class ProfileHeader extends React.Component {
+  render() {
+    return (
+      <div id="profileheader">
```
Comment (khattakdev) [inline client/src/pages/Profile/index.js]:
```suggestion
      <div id={classes.profileheader}>
```

File: client/src/Pages/Profile/index.js [hunk 10]
```diff
@@ -0,0 +1,183 @@
...
+        </section>
+        </div>
+        )
+    }
+}
+
+class UpcomingMeet extends React.Component{
+    render(){
+        return(
+            <div className = {classes.card} id = "upcomingmeeting">
```
Comment (khattakdev) [inline client/src/Pages/Profile/index.js]:
```suggestion
            <div className = {classes.card} id ={classes.upcomingmeeting}>
```

File: client/src/Pages/Profile/index.js [hunk 11]
```diff
@@ -0,0 +1,183 @@
...
+        )
+    }
+}
+
+class ProfileContent extends React.Component {
+    render(){
+        return(
+            <div id="profile">
+            <section className = {classes.cards}>
+                <div className = {classes.card} id = "about">
```
Comment (khattakdev) [inline client/src/Pages/Profile/index.js]:
```suggestion
                <div className = {classes.card} id = {classes.about}>
```

File: client/src/Pages/Profile/index.js [hunk 12]
```diff
@@ -0,0 +1,183 @@
...
+                    </div>
+            </div>
+        )
+    }
+}
+
+class Skills extends React.Component{
+    render(){
+        return(
+            <div className = {classes.card} id = "skills">
```
Comment (khattakdev) [inline client/src/Pages/Profile/index.js]:
```suggestion
            <div className = {classes.card} id ={classes.skills}>
```

File: client/src/Pages/Profile/index.js [hunk 13]
```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import "./index.module.css";
+import classes from "./index.module.css";
+
+class Badges extends React.Component{
+    render(){
+        return(
+            <div className = {classes.card} id = "badge">
```
Comment (khattakdev) [inline client/src/Pages/Profile/index.js]:
```suggestion
            <div className = {classes.card} id ={classes.badge}>
```

**Review by khattakdev — CHANGES_REQUESTED** [review]
You are using `React.Component` for every class.
A better option is to import `Component` once at the top `import React, {Component} from "react" and just use `extends Component`

**Comment by Drewbi** [comment]
Wow @eKhattak Good spots

**Comment by RashikaKarki** [comment]
@eKhattak @Drewbi  I think now it's def good to go. Do give a quick scan and if everything is working well I will merge it.

**Comment by khattakdev** [comment]
> @eKhattak @Drewbi I think now it's def good to go. Do give a quick scan and if everything is working well I will merge it.

You missed that import part.

> You are using `React.Component` for every class.
> A better option is to import `Component` once at the top `import React, {Component} from "react" and just use `extends Component`

**Review by khattakdev — CHANGES_REQUESTED** [review]
Now it looks good!

**Review by khattakdev — APPROVED** [review]

**Pull request merged**